hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/statemachine/background/BlockDeletingService.java

```diff
@@ -220,7 +220,7 @@ private boolean isDeletionAllowed(ContainerData containerData,
           long minReplicatedIndex =
               ratisServer.getMinReplicatedIndex(pipelineID);
           long containerBCSID = containerData.getBlockCommitSequenceId();
-          if (minReplicatedIndex >= 0 && minReplicatedIndex < containerBCSID) {
```

Comment:
Is this intended? `minReplicatedIndex` could be `-1`, thus getting rid of the `minReplicatedIndex >= 0` condition here changes the logic slightly.

cc @szetszwo

Reply:
I agree that it is correct to remove `minReplicatedIndex >= 0`.